solidity_contracts/src/Security/NoDelegateCall.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
uniswapV3 should be imported as a dependency instead of copy paste

Reply:
It is not possible to import it from the uniswapV3 repo as the version is fixed to 0.8.12 
https://github.com/Uniswap/v3-core/blob/0.8/contracts/NoDelegateCall.sol

So let leave it this way